app/src/main/res/values-pt/strings-pt.xml

```diff
@@ -1,496 +1,521 @@
+    <string name="dexcom_share_server_upload">Dexcom Share Server Upload</string>
+    <string name="upload_data_to_dex_servers">Enviar dados para os servidores da Dexcom para que possa usa-los nas apps Dexcom</string>
+    <string name="manage_followers">Gerir Seguidores</string>
+    <string name="manage_existing_followers">Gira os seus seguidores e convide novos.</string>
+    <string name="glucose_calibration_and_other_alerts">Glicémia, Calibração e Outros alertas</string>
+    <string name="alert_volume_profile">Perfil de Volume para Alertas</string>
+    <string name="smart_alerting">Alertas inteligentes</string>
+    <string name="dont_alarm_during_phone_calls">Não alertar durante chamadas</string>
+    <string name="start_snoozed">Começar com Snooze</string>
+    <string name="adjust_all_by_a_factor">Ajustar todos por um fator</string>
+    <string name="reset">Reiniciar</string>
+    <string name="pebble_and_android_wear_options">Opções Pebble e Android Wear</string>
+    <string name="customize_colours">Personalizar Cores</string>
+    <string name="xdrip_plus_color_settings">Definiçoes de cores xDrip+</string>
+    <string name="glucose_values_and_lines">Valor de Glicémia e Linhas</string>
+    <string name="high_glucose_values">Valores Glicémia Hiper</string>
+    <string name="in_range_glucose_values">Valores Glicémia no algo</string>
+    <string name="default_color_selected">Cor pré-definida Selecionado</string>
+    <string name="xdrip_plus_extra_settings">Definições extra xDrip+</string>
+    <string name="example_chart">Gráfico exemplo</string>
+    <string name="treatments_prediction_curves">Tratamentos / Curva preditiva</string>
+    <string name="carbs_per_unit">HC por Uni.</string>
+    <string name="everyday">Todo o dia</string>
+    <string name="insulin_sensitivity_glucose_drop_per_unit">Fator de sensibilidade à insulina: baixa de glicémia por Uni</string>
+    <string name="transmitter_battery">bateria do transmissor</string>
+    <string name="very_low">MUITO BAIXA</string>
+    <string name="low">baixa</string>
+    <string name="space_units">" unidades"</string>
+    <string name="units">unidades</string>
+    <string name="carbs">HC</string>
+    <string name="when">quando</string>
+    <string name="speak_your_treatment">Fale o seu tratamento ex:\nx.x units insulin / xx grams carbs</string>
+    <string name="speech_recognition_is_not_supported">Reconhecimento de fala não suportado</string>
+    <string name="treatment_note">Nota de Tratamento</string>
+    <string name="could_not_export_database">Impossível exportar base de dados :(</string>
+    <string name="exported_to">"Exportada para "</string>
+    <string name="default_to_voice_input_next_time">Pré-definir introdução por voz para a próxima vez</string>
+    <string name="done">Feito</string>
+    <string name="entering_a_calibration_now_will_override_your_previously_entered_calibration">Ao introduzir uma calibração agora irá redefin
```

Comment:
Is estável the correct word here? English is "heading in the right direction", so for a low alert this would be rising and high alert this would be falling. Doesn't estável mean steady or stable?